src/OrchardCore.Modules/OrchardCore.Email/OrchardCore.Email.csproj

```diff
@@ -16,6 +18,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Admin.Abstractions\OrchardCore.Admin.Abstractions.csproj" />
+    <ProjectReference Include="..\..\OrchardCore\OrchardCore.Data.YesSql.Abstractions\OrchardCore.Data.YesSql.Abstractions.csproj" />
```

Comment:
This doesn't seem to be necessary.

Reply:
Hmm, where exactly? Because after I remove this it still builds for me. Not a big deal, just curious.

Reply:
I just tried it and it does not build if we remove it. The `DataMigration` class is coming from the `OrchardCore.Data.YesSql.Abstraction` project

Reply:
Hmm, strange. Anyway.